.github/workflows/news-file.yml

```diff
@@ -4,6 +4,9 @@ on:
   pull_request:
     types: [labeled, unlabeled, opened, reopened, synchronize]
 
+permissions:
+  contents: read
```

Comment:
Is this necessary -- this is only on `pull_request`, which does not get a token with write anyway?

I might have a gap in understanding here, so I'd appreciate context on why this exists.

Reply:
I agree it doesn’t get write token. It is best practice to set permissions. 

That was my reasoning. 

Thanks